### PR TITLE
Slider text function

### PR DESCRIPTION
**Pull request by cchan987**
Merge 2nd
Merge the slider_set_value branch first!

### CODE REVIEW

**Comment by cchan987** [comment]
solves #105

File: nengo_viz/static/viz_slider.js [hunk 1]
```diff
@@ -239,3 +345,9 @@ VIZ.Slider.prototype.layout_info = function () {
     info.max_value = this.scale.domain()[0];
     return info;
 };
+
+VIZ.Slider.prototype.update_value_text = function (slider_index, new_text) {
+    var self = this;
```
Comment (tcstewar) [inline nengo_viz/static/viz_slider.js]: For style consistency, this should be removed and just use ```this``` instead.

If we want to, we could go through all the code and use ```self``` everywhere (although I'd like to see the various reasons for and against that).  However, right now that's not the approach we've been doing, so we should keep the code style consistent.

**Comment by tcstewar** [comment]
This PR breaks the set value capability when there is more than one slider.

To replicate, open up 2d.py and try using the "set value" option in the menu for the slider.

**Comment by cchan987** [comment]
This one is looking good for testing/merging

File: nengo_viz/static/viz_slider.js [hunk 2]
```diff
@@ -386,3 +387,10 @@ VIZ.Slider.prototype.layout_info = function () {
     info.max_value = this.scale.domain()[0];
     return info;
 };
+
+//slider_index: Nat, new_text: Num
+//Rounds to 2 decimal places
+VIZ.Slider.prototype.update_value_text = function (slider_index, new_text) {
```
Comment (tcstewar) [inline nengo_viz/static/viz_slider.js]: ```new_text``` isn't text, so this variable name seems a bit weird.  ```new_value``` would be clearer.

**Pull request merged**